src/pages/Home/Home.test.js

```diff
@@ -0,0 +1 @@
+/* TODO: Add tests specific to component. */
```

Comment:
You should test to make sure it renders correctly, it will be a copy and paste from your other tests.